src/workspace/index.ts

```diff
@@ -104,6 +114,7 @@ function updateScriptsToPackageJson(): Rule {
       "test": "ng test --watch=false",
       "lint:lint": "ng lint",
       "format:lint": "ng lint --fix",
+      "lint:styles": "stylelint \"src/app/**/*.scss\"",
```

Comment:
Also include this in the `build` command.

Reply:
Updated the husky pre-push hook as well!